usr.sbin/devinfo/devinfo.c

```diff
@@ -311,6 +434,11 @@ main(int argc, char *argv[])
 	bool			uflag;
 	char			*path = NULL;
 
+	argc = xo_parse_args(argc, argv);
+	if (argc < 0) {
```

Comment:
how argument count can be lower than zero? program name is counted as an argument as well.

Reply:
xo_parse_args returns -1 on error.

Reply:
Oh that's interesting. I just saw the man page of this function and you're absolutely right!